source/jormungandr/jormungandr/street_network/utils.py

```diff
@@ -59,3 +59,24 @@ def make_speed_switcher(req):
         FallbackModes.ridesharing.name: req['car_no_park_speed'],
         FallbackModes.taxi.name: req['taxi_speed'],
     }
+
+
+PARK_RIDE_VALUES = {'yes'}
```

Comment:
if you want to declare here, please erase this little cute https://github.com/CanalTP/navitia/pull/3228/files#diff-a16a0bc9cd0342e4e2b429a601790fe0R46

Reply:
you're right!